starks/floatingpoint.py

```diff
@@ -0,0 +1,189 @@
+import random
```

Comment:
Could you add a unit test file `test_floatingpoint.py` that adds unit tests for floating point operations? This could be similar to the existing `test_finitefield.py` and other test files

Reply:
Sure, I will add test_floatingpoint.py this week.